tf2/include/tf2/buffer_core.h

```diff
@@ -382,12 +382,16 @@ class BufferCore : public BufferCoreInterface
   std::string allFramesAsStringNoLock() const;
 
   bool setTransformImpl(
-    const tf2::Transform & transform_in, const std::string frame_id,
-    const std::string child_frame_id, const TimePoint stamp,
+    const tf2::Vector3 & origin_in, const tf2::Quaternion & rotation_in,
+    const std::string & frame_id, const std::string & child_frame_id, const TimePoint stamp,
     const std::string & authority, bool is_static);
   void lookupTransformImpl(
```

Comment:
@kyle-basis If I am reading this correctly, the original implementation of `lookupTransformImpl` that uses `tf2::Transform` is a private member function (i.e. not part of the exposed API). I don't see that implementation called anywhere else in the cpp file. Can you confirm?

If this indeed the case shouldn't we be able to remove the `lookupTransformImpl` that uses `tf2::Transform` as no external functions should be calling this implementation?

I always like to prune dead code where we can. 😃